app/src/app.vue

```diff
@@ -29,7 +35,11 @@ import { useAppStore } from '@directus/stores';
 import { User } from '@directus/types';
 import { StyleValue, computed, onMounted, onUnmounted, toRefs, watch } from 'vue';
 import { useI18n } from 'vue-i18n';
+import { useRoute } from 'vue-router';
 import { startIdleTracking, stopIdleTracking } from './idle';
+import PrivateView from './views/private/private-view.vue';
+import PublicView from './views/public/public-view.vue';
+import SharedView from './views/shared/shared-view.vue';
```

Comment:
These are technically globally registered, but I'd like to move away from that in favor of locally importing them for the enhanced DX (type safety through Volar)